### ISSUE TITLE
[Proposal] Добавить EngiCyborg'у способность заменять лампочки.

### ISSUE THREAD

**Issue by Ro2tCrab** [TauCetiStation/TauCetiClassic#6641]
#### Подробное описание проблемы
Chad Дрон способен с нуля построить полностью рабочую лампочку.
#### Что должно было произойти
Virgin EngiBorg собирает лампочку на стене с нуля.
#### Что произошло на самом деле
Дрон унижает киборга.
#### Как повторить
-
#### Дополнительная информация:
-

**Comment by Kortez90** [comment]
Это может делать борг с модулем уборщика.

**Comment by Ro2tCrab** [comment]
Верно, но просить киборга сменить модуль или заставлять других игроков менять лампочки не очень хорошая идея.

**Comment by kalazus** [comment]
Клешня инжборга может подбирать и ставить лампочки, так что собрать лампу с 0, или заменить не так уж и сложно.
